test/test-function.js

```diff
@@ -0,0 +1,12 @@
+
```

Comment:
This file should be in your src folder - only files related to unit testing should be in the test folder, and any functional code should be in the src folder.